course/resources/blurb-rqm.md

```diff
@@ -0,0 +1,3 @@
+We are an interdisciplinary group of graduate students interested in launching a third or fourth year undergraduate course in Reproducible Quantitative Methods (RQM). The curriculum is based on an [open curriculum](https://cbahlai.github.io/rqm-template/) developed by Dr. Christie Bahlai at Michigan State University through a Mozilla Science Lab fellowship. It is specifically designed to meet a growing demand for reproducible, openly accessible, statistically thorough, and well documented science. Although daunting at times, this approach to research presents an opportunity to improve the scientific practice, encourage broader participation in the scientific enterprise, and facilitate meeting the increasingly strict requirements from funding agencies in terms of data accessibility and reproducibility.
```

Comment:
Yes, I use [Markdown Here](http://markdown-here.com/) do convert to html and render it. It works on any rich text input field, including email, really nifty.

What about only "analytically thorough"? Doesn't that imply sounds stats, at least somewhat?